client-app/src/examples/portfolio/GridPanelModel.ts

```diff
@@ -6,7 +6,6 @@ import {PERSIST_MAIN} from './AppModel';
 import {mktValCol, nameCol, pnlCol} from '../../core/columns';
 import {PortfolioPanelModel} from './PortfolioPanelModel';
 import {capitalize} from 'lodash';
-import {Icon} from '@xh/hoist/icon';
 
 export class GridPanelModel extends HoistModel {
     @managed
```

Comment:
Now that we no longer need to externally observe its collapsed state, I think we could further clean this up by replacing this PanelModel with an inline `modelConfig: {}` in the component. This is a major benefit of supporting the collapsed title switching internally.

Reply:
Good point! I've made the change.